desktop/libs/hadoop/src/hadoop/fs/upload.py

```diff
@@ -116,7 +116,18 @@ def remove(self):
         LOG.exception('Failed to remove temporary upload file "%s". Please cleanup manually: %s' % (self._path, ex))
 
   def write(self, data):
-    self._file.write(data)
+    try:
+      self._file.write(data)
+    except WebHdfsException as e:
+      LOG.exception("WebHdfsException: " + str(e))
+      if "append() Not implemented by the OzoneFileSystem FileSystem implementation" in e.message:
+        self.create(data)
+  
+  def create(self, data):
+    # If temp file exist, we are deleting it
+    if self._fs.exists(self._path):
+      self._fs._delete(self._path)
+    self._file.create(data)
```

Comment:
Question: If we are going with this approach for file upload for Ozone via HDFS connector, then we also need to increase chunk size for all filesystems right? or was any other approach discussed for HDFS connector??